src/features/Team/TeamDescription.tsx

```diff
@@ -17,29 +19,111 @@ const TeamDescription: React.FC<ITeamDescriptionProps> = (
   props: ITeamDescriptionProps
 ) => {
   return (
-    <MaxWidthBox maxWidth={'400px'} mx={[5, 'auto']}>
-      <H1 fontSize={'30px'} marginTop={'0px'} marginLeft={'0px'}>
-        Your Team
-      </H1>
-      <Flex flexDirection={'column'}>
-        <Box>
-          <TeamView team={props.team} members={props.members} />
-        </Box>
-        <Box mt={'15px'}>
-          <Flex justifyContent={'center'}>
-            <Box>
-              <Button
-                onClick={props.onLeaveTeam}
-                isLoading={props.isLeavingTeam}
-              >
-                Leave team
-              </Button>
-            </Box>
-          </Flex>
-        </Box>
-      </Flex>
-    </MaxWidthBox>
+    <>
+      <div className="team-code-container">
+        <div className="label">Team code</div>
+        <div className="info-text">Share this code with your team members to let them join:</div>
+        <div className="team-code">
+          <ClipboardComponent
+            value={props.team.name}
+            onSuccess={onCopied}
+            onError={onCopyFailed}
+          />
+        </div>
+      </div>
+
+      <div className="team-members-container">
+        <div className="label">Members</div>
+        <div className="members">
+          {props.members.map((member, index) => {
```

Comment:
Nit: this anonymous function should be its own component, which renders a team member

Reply:
Fixed